docs/start/quickstart.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Quick Start
```

Comment:
Nit: I think quickstart is a valid word? (no space) if we want to use that

Reply:
ok